src/client/client.ts

```diff
@@ -435,7 +435,14 @@ export class _ClientImpl<G extends any = any> {
     // Secrets are normally stripped on the server,
     // but we also strip them here so that game developers
     // can see their effects while prototyping.
-    const G = this.game.playerView(state.G, state.ctx, this.playerID);
+    let G;
```

Comment:
It might be cleaner to use a ternary operator to avoid reassignment here:

```js 
const G = this.multiplayer
  ? state.G
  : this.game.playerView(...)
```